### DISCUSSION TITLE
Unable to build apk/aab file (AIR SDK 33.1.1.633)

### DISCUSSION THREAD

**Discussion by idzdigital** [airsdk/Adobe-Runtime-Support#1265] [General]
![image](https://user-images.githubusercontent.com/39980526/137312697-199cef6d-9d0f-4945-9c8a-2d3f33719800.png)
Hi, 
I am using AIR SDK 33.1.1.633 and min target sdk is 21 and max is 30 ,while creating app bundle from publish settings ,I am getting this error, Can you please help with this. 
Error creating files.Unknown error. Unexpected failure: Unable to rename lib -> jnilibs java.io.|OException: Unable to rename lib -> jnilibs  at com.adobe.air.apk AABOutputStream.addCaptiveRuntimeLibs{ ‘AABOutputStream.java)  at com.adobe.airapk AABOutputStream.addpplicationDescript or(AABOutputStream,java)  at com.adobe.air ApplicationPackageraddSpecialiles(Applicatio nPackager.java)  at com.adobe.air-ApplicationPackager.createPackage(Application Packager.java)  at com.adobe.ai.apk AABPackager.createPackage(AABPackager ja va)  at comadobe.air ADT.parseArgsAndGo(ADT Java}  at comadobe air ADT.run(ADT java)  at comadobe air ADT main(ADT,java)
I have also attached adt.cfg file image below, 
Thanks in advance.
![image](https://user-images.githubusercontent.com/39980526/137314122-c8e6224c-4285-4c3b-9071-9935758acfe1.png)

**Comment by ajwfrost** [comment]
Something perhaps is holding open a file reference to the jniLibs folder ..? Are you able to close down any other apps that are accessing any folders within the AndroidStudioProject tree, including command prompts or similar, and re-try? To be sure, you should delete that AndroidStudioProject folder manually before starting..

We can put in a bit of a check to ensure that the clean-up works perhaps, to make it a better warning message rather than throwing this exception.

thanks
